airflow/operators/python.py

```diff
@@ -421,7 +421,7 @@ def __deepcopy__(self, memo):
         return super().__deepcopy__(memo)
 
     def _execute_python_callable_in_subprocess(self, python_path: Path, tmp_dir: Path):
-        op_kwargs: dict[str, Any] = {k: v for k, v in self.op_kwargs.items()}
+        op_kwargs: dict[str, Any] = dict(self.op_kwargs.items())
```

Comment:
```suggestion
        op_kwargs: dict[str, Any] = self.op_kwargs.copy()
```

Reply:
op_kwargs type is `Mapping` that doesn't have `copy()` method

Reply:
Then `dict(self.op_kwargs)` should work, I think.